Library/Formula/fpc.rb

```diff
@@ -25,11 +25,15 @@ def install
     system "make", "install", "PP=#{fpc_compiler}", "PREFIX=#{prefix}"
 
     bin.install_symlink lib/"#{name}/#{version}/ppcx64"
+
+    # generate a default fpc.cfg to facilitate search paths
+    system "./utils/fpcmkcfg/fpcmkcfg", "-p", "-d", "basepath=#{Formula["fpc"].opt_lib}/fpc/" + `./compiler/ppc1 -iW`.chomp, "-o", "#{Formula["fpc"].opt_prefix}/etc/fpc.cfg"
```

Comment:
I'm not sure I follow. To recap, in order to write fpc.cfg you need to find wherever the final fpc is going to be, then go above one level and then cd to etc. Since fpc could be placed in different places the basepath has to be adjusted every time, and you need to specify it at command line.

So, would this be the right approach? `system "./utils/fpcmkcfg/fpcmkcfg", "-p", "-d", "basepath=#{prefix}/lib/fpc/" + `./compiler/ppc1 -iW`.chomp, "-o", "#{prefix}/etc/fpc.cfg"`